apps/e2e/ag-grid-storybook/src/app/ag-grid/ag-grid-stories.component.html

```diff
@@ -1,7 +1,36 @@
-<sky-ag-grid-wrapper *ngIf="isActive$ | async">
-  <ag-grid-angular
-    [gridOptions]="gridOptions"
-    [rowData]="items"
-  ></ag-grid-angular>
-</sky-ag-grid-wrapper>
```

Comment:
This file has been completely refactored to test multiple grids on one screenshot.